sdk/common/src/main/resources/log4j2.xml

```diff
@@ -4,15 +4,14 @@
         <Property name="PATTERN">%-5level %d [%t] %c:%M(%L): %m%n</Property>
     </Properties>
 
-
     <Appenders>
         <Console name="STDOUT" target="SYSTEM_OUT">
             <PatternLayout pattern="${PATTERN}"/>
         </Console>
     </Appenders>
 
     <Loggers>
-        <Root level="INFO">
+        <Root level="${env:FRAMEWORK_LOG_LEVEL}">
```

Comment:
This should have a default specified, in case a framework developer effs up: https://stackoverflow.com/questions/22028399/log4j2-property-substitution-default